packages/components/src/components/loaders/Spinner.tsx

```diff
@@ -20,7 +20,7 @@ const Wrapper = styled.div<FluidSpinnerProps>`
         box-sizing: border-box;
         width: ${({ size }) => `${size}px`}; /* change to 1em to scale based on used font-size */
         height: ${({ size }) => `${size}px`}; /* change to 1em to scale based on used font-size */
-        border: ${({ strokeWidth }) => (strokeWidth ? `${strokeWidth}px` : '1px')} solid transparent; /* change to 0.1em to scale based on used font-size */
+        border: ${({ strokeWidth }) => `${strokeWidth}px`} solid transparent; /* change to 0.1em to scale based on used font-size */
```

Comment:
nitpick, but I would keep here the default value, because type says that it's optional or fix the type for Wrapper props
```suggestion
        border: ${({ strokeWidth }) => `${strokeWidth ?? 2}px`} solid transparent; /* change to 0.1em to scale based on used font-size */
```

Reply:
the default value is in the components params

Reply:
I know, but Wrapper is another component... but it's a nitpick... feel free to ignore

Reply:
i left any in-depth refactoring for later.